helm/rook-operator/values.yaml

```diff
@@ -349,7 +359,7 @@ discoverDaemonUdev:
 # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
 enableOBCWatchOperatorNamespace: true
 
-admissionController: {}
+admissionController:
```

Comment:
hmm will it not fail if we leave an object literal empty?

Reply:
well i wondered that too, but this is pulled directly from the upstream 1.6.8 release https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml#L362

:man_shrugging:

Reply:
nope it looks like it should work fine this way https://github.com/giantswarm/rook-operator-app/blob/main/helm/rook-operator/templates/deployment.yaml#L269

Reply:
ok I would swear I hit that in the past 🙈